appsec-kit-demo-v24-spring/pom.xml

```diff
@@ -58,12 +56,12 @@
         <dependency>
             <groupId>com.vaadin</groupId>
             <artifactId>vaadin-core</artifactId>
-            <version>${flow.version}</version>
+            <version>${flow24.components.version}</version>
         </dependency>
         <dependency>
             <groupId>com.vaadin</groupId>
             <artifactId>vaadin-spring-boot-starter</artifactId>
-            <version>${flow.version}</version>
+            <version>${flow24.components.version}</version>
```

Comment:
Shouldn't this be `flow24.version`?

Reply:
actually no, flow components follow the same version than platform, but flow could be different at some point, so for instance we could have platform-24.2.7 flow-components-24.2.7 but flow-24.2.5 because there were changes in webcomponents but not in flow